velox/functions/sparksql/Arithmetic.h

```diff
@@ -262,6 +262,16 @@ struct Log1pFunction {
   }
 };
 
+
+struct Expm1Function {
+  FOLLY_ALWAYS_INLINE void call(double& result, double a) {
+    // The std::expm1 is more accurate than the expression std::exp(num) - 1.0 if num is close to zero.
+    // Spark use java StrickMath we need to keep it as below.
```

Comment:
Is this implementation equivalent to `StrickMath` used by Spark?

Reply:
Same way. And I've checked the corner cases between both sides and all are the same.
As I don't find the source code for the StrictMath, the check for the implementation of StrictMatch and expm1 in cpp is not done, while the comparison between the Spark and this PR is the same.